### PR TITLE
Integrated with optimized go-trees package

### PR DESCRIPTION
**Pull request by vasili-v**

### CODE REVIEW

File: pdp/content_test.go [hunk 1]
```diff
@@ -617,7 +617,7 @@ func TestLocalContentStorage(t *testing.T) {
 	} else {
 		path := []Expression{
 			MakeStringValue("key"),
-			MakeDomainValue(domain.WireNameLower("\x07example\x03gov\x00"))}
```
Comment (yuewko) [inline pdp/content_test.go]: So, we don't support "wire" format anymore?

Comment (vasili-v) [reply pdp/content_test.go]: Yep. Actually, it ins't big deal to add it too. But I decided to not do it as anyway wire require some transformations like changing case and order of labels.

Comment (yuewko) [reply pdp/content_test.go]: OK. Thanks.

**Comment by vasili-v** [comment]
Updated go-trees package

**Pull request merged**